js/adapt-contrib-tutor.js

```diff
@@ -7,7 +7,33 @@ import TUTOR_TYPE from './TUTOR_TYPE';
 class Tutor extends Backbone.Controller {
 
   initialize() {
-    this.listenTo(Adapt, 'questionView:showFeedback', this.onQuestionViewShowFeedback);
+    this.listenTo(Adapt, {
+      'componentView:postRender': this.onComponentViewPostRender,
+      'questionView:showFeedback': this.onQuestionViewShowFeedback
+    });
+  }
+
+  onComponentViewPostRender(view) {
+    const model = view.model;
+    if (!model.isTypeGroup('question')) return;
+    const config = model.get('_tutor');
+    if (!config) return;
+    const type = TUTOR_TYPE(config?._type.toUpperCase());
```

Comment:
```suggestion
    const type = TUTOR_TYPE(config._type?.toUpperCase());
```
config is already checked on line 20. It is more likely the `_type` needs guarded for the function call

Reply:
Good catch. Thanks.